README.md

```diff
@@ -1,2 +1,4 @@
 # ble-peripheral
```

Comment:
It is unfortunate for sure and I am aware of the reused wording, but the big catch 22 is that the BLE spec is very clear that the device role being implemented by this code is called "peripheral" (the "server" side), contrasted with the device role of "central" (the "client" side).  Bluetooth is an obtuse and bloated standard with so many unusual terms that it can be very confusing to navigate and understand without strong and consistent implementation wording that matches the spec.   I sincerely believe we're better off catering to Bluetooth terms than to Rust embedded terms to avoid confusion, especially given that this code has applicability beyond the embedded-hal ecosystem.

Maybe a good compromise though would be to use redundant local naming and call it BlePeripheral or something.  Or PeripheralRole perhaps.